src/utilities/utilities.ts

```diff
@@ -39,6 +39,9 @@ export async function execFile(
         `Exec: ${executable} ${args.join(" ")}`,
         folderContext.name
     );
+    if (configuration.runtimePath.length > 0) {
+        options.env = { ...options.env, ...swiftRuntimePathEnv(options.env) };
+    }
```

Comment:
`options.env` may be undefined here, if that is the case you should use `process.env` as the basis for runtime path construction

Reply:
If `options.env` is undefined, `swiftRuntimePathEnv()` will use the default value, which is `process.env`.